src/components/editor/EditAuthorsForm.jsx

```diff
@@ -1,80 +1,37 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import BaseComponent from 'lib/BaseComponent';
 import _ from 'lodash';
 
 export default class EditAuthorsForm extends BaseComponent {
   constructor(props) {
     super(props);
     this.safeSetState({
-      // Key is slug, value is name
-      authorsAdded: {},
-      // An object where key is slug and value is whether it's deleted or not
-      authorsDeleted: _.reduce(this.props.data, (cur, next) => {
-        cur[next.slug] = false;
-        return cur;
-      }, {})
+      addAuthorValue: ""
     });
     this.handleAddAuthor = this.handleAddAuthor.bind(this);
   }
 
   componentDidUpdate() {
-    this.props.onChange(this.state.authorsAdded, this.state.authorsDeleted);
+    this.props.onChange(ReactDOM.findDOMNode(this).parentNode);
```

Comment:
I use it at EditorArticleController line 40? It's how I access all the values of the main form.

And what is it exactly you mean with the fragility? You might very well be right, I just don't quite understand what you mean.

And if it's bad form (it did also ring some red flags as well as I know we're trying to avoid using the DOM, as it's not reacty enough, right? ), do you have suggestions for a better way? Because I need a way to access the form values from the callback, that's my biggest obstacle, which I used the DOM calls to fix.